src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int diggerSize;
+    private int diggerCapacity;
```

Comment:
Consider using private access modifiers for the custom fields `diggerSize` and `diggerCapacity` to ensure encapsulation. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters).